### ISSUE TITLE
More Realistic Light Pollution

### ISSUE THREAD

**Issue by Noobly-Walker** [Povstalec/StellarView#32]
Every torch above sea level increases the light pollution in a chunk by 1, up to the point where the only celestial bodies visible from there at night is the moon. Light pollution spreads to neighboring chunk in the same way light spreads to neighboring blocks, where each chunk orthogonality adjacent to the most polluted one has 1 less light pollution.

Record light pollution for each chunk and save it to build a light pollution map. If light pollution updates due to some light source being discovered in new chunks (or even old ones), the light pollution map updates smoothly, so the stars don't blink out.

**Comment by Povstalec** [comment]
Impossible to do on client

**Comment by Noobly-Walker** [comment]
I was more thinking some kind of database.

**Comment by Joe-Moe** [comment]
Is there a config option to disable light pollution?

**Comment by Povstalec** [comment]
> Is there a config option to disable light pollution?

Yes, you can find it in the General Config Screen under the name Light Pollution.
